src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { UserInfo } from '../UserInfo';
+import { getUsers } from '../../api/api2';
```

Comment:
you didn't add your file api2, and, actually you should remane it, api2 - bad naming